deployments/helm-chart/values.yaml

```diff
@@ -1,63 +1,176 @@
   affinity: {}
+
+  ## The number of replicas of the Ingress controller deployment.
   replicaCount: 1
+
+  ## A class of the Ingress controller. The Ingress controller only processes Ingress resources that belong to its class
+  ## i.e. have the annotation "kubernetes.io/ingress.class" equal to the class.
+  ## Additionally, the Ingress controller processes Ingress resources that do not have that annotation which can be disabled by setting the "-use-ingress-class-only" flag.
   ingressClass: nginx
+
+  ## Ignore Ingress resources without the "kubernetes.io/ingress.class" annotation.
   useIngressClassOnly: false
+
+  ## Namespace to watch for Ingress resources. By default the Ingress controller watches all namespaces.
   watchNamespace: ""
+
+  ## Add a location "/nginx-health" to the default server. The location responds with the 200 status code for any request.
+  ## Useful for external health-checking of the Ingress controller.
   healthStatus: false
+
   nginxStatus:
+    ## Enable the NGINX stub_status, or the NGINX Plus API.
     enable: true
+
+    ## Set the port where the NGINX stub_status or the NGINX Plus API is exposed.
     port: 8080
+
+    ## Whitelist IPv4 IP/CIDR blocks to allow access to NGINX stub_status or the NGINX Plus API. Separate multiple IP/CIDR by commas.
     allowCidrs: "127.0.0.1"
+
   service:
+    ## Creates a service to expose the Ingress controller pods.
     create: true
+
+    ## The type of service to create for the Ingress controller.
     type: LoadBalancer
+
+    ## The externalTrafficPolicy of the service. The value Local preserves the client source IP.
     externalTrafficPolicy: Local
+
+    ## The annotations of the Ingress controller service.
     annotations: {}
+
+    ## The static IP address for the load balancer. Requires controller.service.type set to LoadBalancer. The cloud provider must support this feature.
     loadBalancerIP: ""
+
+    ## The list of external IPs for the Ingress controller service.
     externalIPs: []
+
+    ## The IP ranges (CIDR) that are allowed to access the load balancer. Requires controller.service.type set to LoadBalancer. The cloud provider must support this feature.
     loadBalancerSourceRanges: []
+
     httpPort:
+      ## Enables the HTTP port for the Ingress controller service.
       enable: true
+
+      ## The HTTP port of the Ingress controller service.
       port: 80
+
+      ## The custom NodePort for the HTTP port. Requires controller.service.type set to NodePort.
       nodePort: ""
+
     httpsPort:
+      ## Enables the HTTPS port for the Ingress controller service.
       enable: true
+
+      ## The HTTPS port of the Ingress controller service.
       port: 443
+
+      ## The custom NodePort for the HTTPS port. Requires controller.service.type set to NodePort.
       nodePort: ""
+
   serviceAccount:
+    ## Creates a service account. Required for RBAC.
+    create: true
```

Comment:
this is not used in the manifests. consider removing it

Reply:
Nice catch, will do.